webpack.config.js

```diff
@@ -0,0 +1,49 @@
+var path = require("path")
+var webpack = require("webpack")
+var BundleTracker = require('webpack-bundle-tracker')
+
+module.exports = {
+  context: __dirname,
+
+  // app import chain starts here
+  entry: [
+    'webpack-dev-server/client?http://localhost:3000',
+    'webpack/hot/only-dev-server',
+    './frontend/app/app',
+  ],
+  // location of written bundled files
+  output: {
+    path: path.resolve('./frontend/bundles/'),
+    filename: '[name]-[hash].js',
+    publicPath: 'http://localhost:3000/static/bundles/',
+  },
+
+  plugins: [
+    new webpack.HotModuleReplacementPlugin(),
+    new BundleTracker({path: __dirname, filename: './webpack-stats.json'}),
+    new webpack.NoErrorsPlugin(),
+  ],
+
+  module: {
+    loaders: [
+      { test: /\.jsx?$/,
+        exclude: /node_modules/,
+        loaders: ['babel-loader'],
+      },
+      { test: /\.scss$/,
+        loaders: ['style', 'css', 'sass']
+      },
+      { test: /\.css$/,
+        loaders: ['style', 'css']
+      },
+      { test: /\.(png|jpg|jpeg|gif|svg|woff|woff2|ttf|eot)$/,
+        loader: 'file'
+      }
```

Comment:
This part of the webpack config with the file loader will end up being split into using both file loader and url loader after reading/understanding how they work ie http://survivejs.com/webpack/loading-assets/loading-images/. For now i just used this loader for everything so that i could get react-bootstrap working.